.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: java
+sudo: false
```

Comment:
Travis gave me a nice big warning that the build was running on the legacy infrastructure, this removes the warning and makes the builds faster too